team/tests/org.eclipse.compare.tests/src/org/eclipse/compare/tests/DocLineComparatorTest.java

```diff
@@ -258,15 +259,67 @@ public boolean canCacheFilteredRegions() {
 		for (int i = 0; i < docs.length; i++)
 			for (int j = i + 1; j < docs.length; j++)
 				for (ICompareFilter[] filter : filters) {
-					l = new DocLineComparator(docs[i], null, false, filter, 'L');
-					r = new DocLineComparator(docs[j], null, false, filter, 'R');
+					l = new DocLineComparator(docs[i], null, false, filter, 'L', Optional.empty());
+					r = new DocLineComparator(docs[j], null, false, filter, 'R', Optional.empty());
 					Assert.assertFalse(l.rangesEqual(0, r, 0));
-					l = new DocLineComparator(docs[i], null, true, filter, 'L');
-					r = new DocLineComparator(docs[j], null, true, filter, 'R');
+					l = new DocLineComparator(docs[i], null, true, filter, 'L', Optional.empty());
+					r = new DocLineComparator(docs[j], null, true, filter, 'R', Optional.empty());
 					Assert.assertTrue(l.rangesEqual(0, r, 0));
 				}
 	}
 
+	@Test
+	public void whiteSpaceInStringLiteralWithEmptyIgnoreWhitespaceContributor() {
```

Comment:
The method name does not state the expectation. If I got it right this method checks that whitespace is handled as of today if no whitespace contributor is suplied. Maybe

```suggestion
	public void noWhitespaceContributorSupplied_whitespaceInStringLiteralNotIgnored() {
```

is a better name.

Reply:
changed with 9376205, correct name is noWhitespaceContributorSupplied_whitespaceInStringLiteralIgnored